dist/src/test/java/org/wildfly/dist/subsystem/xml/AbstractValidationUnitTest.java

```diff
@@ -80,6 +80,7 @@
         EXCLUDED_SCHEMA_FILES.add("jboss-ejb-container-interceptors_1_0.xsd");
         EXCLUDED_SCHEMA_FILES.add("jboss-ejb-delivery-active_1_0.xsd");
         EXCLUDED_SCHEMA_FILES.add("jboss-ejb-delivery-active_1_1.xsd");
+        EXCLUDED_SCHEMA_FILES.add("jboss-ejb-delivery-active_1_2.xsd");
```

Comment:
Another file also needs this same change:

https://github.com/wildfly/wildfly/blob/master/dist-legacy/src/test/java/org/wildfly/dist/subsystem/xml/AbstractValidationUnitTest.java#L82